Reservia.html

```diff
@@ -0,0 +1,218 @@
+                    <figcaption class="figcaption1">Hotel les mouettes</figcaption>
+                   <p>Nuit à partir de <span>76 €</span></p>
+                    <p><img id="star" src="Reservia_files/star4.PNG" alt=""></p>
+              <!--  </figure> -->
+                </div>
+                <div id="sousbox1-4">
+                   <!-- <figure>-->
+                       <img class="imgbox1"id="hostimg4" src="Reservia_files/hostimg4.jpg" alt="">
+                    <!--<img class="imgbox1" src="Reservia_files/hostimg4.jpg" alt="">-->
+                    <figcaption class="figcaption1">Hotel de la mer</figcaption>
+                    <p>Nuit à partir de <span>46 €</span></p>
+                    <p><img id="star" src="Reservia_files/star3.PNG" alt=""></p>
+                <!--</figure> -->
+                   
+                </div>
+                <div id="sousbox1-5">
+                   <!-- <figure>-->
+                  <img class="imgbox1" id="hostimg5"src="Reservia_files/hostimg5.jpg" alt="">
+                    <figcaption class="figcaption1">Auberge le Panier</figcaption>
+                    <p>Nuit à partir de <span>23 €</span></p>
+                    <p><img id="star" src="Reservia_files/star4.PNG" alt=""></p>
+                <!--</figure> -->
+                </div>
+                <div id="sousbox1-6">
+                  <!--  <figure>-->
+                   <img class="imgbox1" id="hostimg6" src="Reservia_files/hostimg6.jpg" alt="">
+                    <figcaption class="figcaption1">Hotel chez Amina</figcaption>
+                   <p>Nuit à partir de <span>96 €</span></p>
+                    <p><img id="star" src="Reservia_files/star5.PNG" alt=""></p>
+                <!--</figure> -->
+                  
+                </div>
+            </div>
+      
+            
+            <div id="box2"><!-- Two box -->
+               <div id="sousbox2-1">
+                   <img id="hostimg7" class="imgbox2" src="Reservia_files/hostimg7.jpg" alt="">
+                    <figcaption class="figcaption2" >Hotel le soleil du matin</figcaption>
+                    <p >Nuit à partir de <span>96 €</span></p>
+                    <p><img id="star" src="Reservia_files/star5.PNG" alt=""></p>
+                  
+                </div>
+               <div id="sousbox2-2">
+                    <img id="hostimg8"class="imgbox2" src="Reservia_files/hostimg8.jpg" alt="">
+                    <figcaption class="figcaption2" id="idFigcaption1">Au coeur de l'eau<br>Chambre d'hotes</figcaption>
+                </div>
+                <div id="sousbox2-3">
+                    <img id="hostimg9" class="imgbox2"src="Reservia_files/hostimg9.jpg" alt="">
+                    <figcaption class="figcaption2"id="idFigcaption2">Hotel tout bleu et blanc</figcaption>
+                </div>
+            </div>
+
+
+
+        </section>
+   <!--</br>
```

Comment:
useless code, take time to remove everything that is not needed anymore. if not this will become legacy code and will become a nightmare to handle for the next developer to work on this page (of course in this case nobody else will work on it but hey that's a good habbit to take to remove deprecated code that is not used anymore)

Reply:
the useless code has been erased in order to have a clean code.